### PR TITLE
add slack icon to website

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

**Comment by gdams** [comment]
staging: http://staging.adoptopenjdk.net/

**Comment by tellison** [comment]
Looks fine to me.

FYI @karianna was going to subscribe to the archive service and get us a custom domain name, which will require a small tweak here.

**Comment by karianna** [comment]
I'm just waiting for some Paypal access to the LJC and I'll have the custom domain and the whole archive - hope to resolve in the next day or two.

File: about.html [hunk 1]
```diff
@@ -109,6 +109,13 @@ <h2 class="inline-block zero-margin">Submit an Issue&nbsp;<i class="fa fa-arrow-
 <footer>
   <div class="margin-auto">
     <div class="footer-icon align-center inline-block">
+      <a href="https://adoptopenjdk.slackarchive.io/" target="_blank" class="light-link no-underline">
+        <img src="dist/assets/slack.png" alt="" class="footer-logo">
```
Comment (karianna) [inline about.html]: I'll fix this URL once I get the custom domain

**Comment by gdams** [comment]
can we not just save the money and do a http redirect from our DNS?

**Comment by tellison** [comment]
I've added comments to the existing issue about [Slack archiving](https://github.com/AdoptOpenJDK/openjdk-build/issues/12), but in summary we get a URL and can import old messages.  A simple redirect would still show as slackarchive.io.

**Comment by karianna** [comment]
They've enabled the custom domain (slackarchive.adoptopenjdk.net), so we can cname the DNS to adoptopenjdk.slackarchive.io (I think that's right) and we should be sorted.  @gdams or @tellison are you able to do the CNAME?

**Comment by karianna** [comment]
Oh and I've sent them the old message export, so they'll import that shortly.

**Comment by gdams** [comment]
What am I changing the DNS to? it is unclear here

**Comment by karianna** [comment]
@gdams I've asked for some clarification.

**Pull request merged**